package.json

```
@@ -5,6 +5,7 @@
     "dev-server-guest": "encore dev-server --hot --host 0.0.0.0 --port 8001",
     "dev-server-host": "encore dev-server --hot --port 8002",
     "dev-server-gitpod": "encore dev-server --hot --port 8002 --public $(gp url 8002) --disable-host-check",
+    "prettier": "prettier --write app/Resources/**/*.jsx",
```

Please change also include JS files. The JS files have not been run through prettier before -> commit the prettified versions as well. In addition, I would like to have a second `lint` command to check whether the files are formatted correctly and rename the formatting command to `format`:
```suggestion
    "format": "prettier --write app/Resources/**/*.{js,jsx}",
    "lint": "prettier --check app/Resources/**/*.{js,jsx}",
```

---

I like this change a lot more, didnt know about the commit hook, added those lines as requested https://github.com/AdventureLookup/AdventureLookup/pull/327/commits/86c82016f088f28132e069f57517e8bab234e29d